railties/lib/rails/application/configuration.rb

```diff
@@ -350,14 +350,13 @@ def enable_reloading=(value)
       end
 
       def read_encrypted_secrets
-        Rails.deprecator.warn(`config.read_encrypted_secrets is deprecated and will be removed in Rails 7.3.`)
+        Rails.deprecator.warn("config.read_encrypted_secrets is deprecated and will be removed in Rails 7.3.")
       end
 
       def read_encrypted_secrets=(value)
-        Rails.deprecator.warn(`config.read_encrypted_secrets is deprecated and will be removed in Rails 7.3.`)
+        Rails.deprecator.warn("config.read_encrypted_secrets is deprecated and will be removed in Rails 7.3.")
```

Comment:
```suggestion
        Rails.deprecator.warn("`config.read_encrypted_secrets` is deprecated and will be removed in Rails 7.3.")
```

Reply:
Thanks! Applied separately to keep it a single-commit PR